docs/admin_api/account_validity.md

```diff
@@ -1,5 +1,7 @@
 # Account validity API
 
+**Note:** This API is disabled when MSC3861 is enabled. [See #15582](https://github.com/matrix-org/synapse/pull/15582)
```

Comment:
I think linking off to the pull request is likely not super helpful for end-users.

It is a bit odd though because this is an experimental feature which we don't *really* document. I'm not quite sure what to do about this. (It *does* seem valuable...)

Reply:
Agree with the sentiment that documenting experimental is quite unusual, but equally I worry we're expecting people to start using OIDC with Element X imminently and need to document what may break.

I mean, in an ideal world we'd have gone through and made OIDC stable before asking people to use it in production.